data/benefits/javascript/intercommunalite_metropole_lyon-pret-d-un-velo-freevelo-v.yml

```
@@ -0,0 +1,37 @@
+label: prêt d’un vélo Freevélo’v
+institution: intercommunalite_metropole_lyon
+description: La métropole de Lyon prête un vélo Freevélo’v aux jeunes âgés de 18
+  à 25 ans pour une durée de 3, 6, 9 ou 12 mois. Le prêt est renouvelable
+  jusqu’à 2 ans.
+prefix: le
+conditions_generales:
+  - type: age
+    operator: ">="
+    value: 18
+  - type: age
+    operator: <=
+    value: 25
+  - type: epcis
+    values:
+      - "20004697700019"
```

J'ai un peu galéré à afficher l'aide, ça m'a l'air de fonctionner avec ce code là plutôt:
```suggestion
      - "200046977"
```